ui/imports/shared/status/StatusChatInput.qml

```diff
@@ -144,7 +145,7 @@ Rectangle {
 
     function checkTextInsert() {
         if (emojiSuggestions.visible) {
-            emojiSuggestions.addEmoji();
+            replaceWithEmoji(extrapolateCursorPosition(), emojiSuggestions.shortname, emojiSuggestions.unicode);
```

Comment:
So this looks like a change in logic here...

Is this still part of dynamic scoping or does this actually do something else?
If this happens to be a fix or something, it'd be worth extracting this into its own commit.